opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/_view_instrument_match.py

```diff
@@ -52,12 +52,20 @@ def __init__(
         )
         if not isinstance(self._view._aggregation, DefaultAggregation):
             self._aggregation = self._view._aggregation._create_aggregation(
-                self._instrument, None, 0
+                self._instrument,
+                None,
+                self._view._exemplar_reservoir_factory,
+                0,
```

Comment:
Might need to [rebase](https://github.com/open-telemetry/opentelemetry-python/pull/4137)